src/entities/offerings.ts

```diff
@@ -381,12 +418,24 @@ const toPackage = (
 };
 
 export const toOffering = (
+  isCurrentOffering: boolean,
   offeringsData: OfferingResponse,
   productDetailsData: { [productId: string]: ProductResponse },
+  targetingResponse?: TargetingResponse,
 ): Offering | null => {
+  const presentedOfferingContext: PresentedOfferingContext = {
+    offeringIdentifier: offeringsData.identifier,
+    targetingContext:
+      isCurrentOffering && targetingResponse
+        ? {
+            ruleId: targetingResponse.rule_id,
+            revision: targetingResponse.revision,
+          }
+        : null,
```

Comment:
This is probably a stupid question, but why do we only set `targetingContext` for the current offering?

Reply:
Targeting works by modifying the `current` offering depending on certain parameters (country, sdk version,....). So the targeting information is only relevant to that offering. The developer can always choose to purchase a package in a different offering, but in that case, the targeting information doesn't matter, since the rules applied didn't affect anything.

There is one thing to note that is different here vs the mobile SDKs. That is, when accessing the `current` offering, not through the `current` accessor but through the `all` dictionary, we are adding targeting information in `purchases-js`, but not in the mobile SDKs. This is needed to support the hosted checkout page, but is a distinction we should be aware of. cc @joshdholtz